### ISSUE TITLE
page privacy policy in footer?

### ISSUE THREAD

**Issue by biekemaex** [inbo/craywatch#195]
Do we need to include the privacy policy in the main menu or can it also be a link somewhere in the footer? https://craywatch.inbo.be/privacy/ (I will still add some links to the body of text)

**Comment by peterdesmet** [comment]
Option 1:

<img width="864" alt="Screenshot 2024-06-05 at 10 40 09" src="https://github.com/inbo/craywatch/assets/600993/09ef2d3a-7595-4a49-b3b1-c9beeec4554b">

Option 2:

<img width="953" alt="Screenshot 2024-06-05 at 10 41 22" src="https://github.com/inbo/craywatch/assets/600993/b8e20b6e-8321-46ad-83bd-0da1d6d978c1">

Is there a plan to add more links to the footer?

**Comment by peterdesmet** [comment]
PS: `Theme: Petridish` can be removed

**Comment by biekemaex** [comment]
I don't think we will be adding more links. Maybe option 1 so no one can suggest we tried to bury it somewhere? :)

**Comment by peterdesmet** [comment]
We can also do option 3 (in case option 1 looks like it's a privacy policy of ANB):

<img width="859" alt="Screenshot 2024-06-05 at 10 59 41" src="https://github.com/inbo/craywatch/assets/600993/29680b9e-d494-44a8-8fc3-35cd9adb5414">

**Comment by biekemaex** [comment]
that works!
